licenses/BSL.txt

```diff
@@ -0,0 +1,98 @@
+Business Source License 1.1
+
+Parameters
+
+Licensor:             Rocicorp, LLC
+Licensed Work:        repc 0.10.1
+                      The Licensed Work is (c) 2020 Rocicorp, LLC.
+Additional Use Grant: You may use the Licensed Work in a commercial offering
+                      with less than 100 client instances, or in a
+                      non-commercial offering with less than 10,000 client
```

Comment:
10k clients is very roughly on the order of $100/month to support at the diffserver, not counting any time we spend on them.

Reply:
There is actually a technical distinction here. This license just covers legal access to the *code* -- not to the diff server. That is a different question.

Part of me just feels like the code should be completely free for non-commercial use, but 10k feels like a conservative start.

Reply:
> Part of me just feels like the code should be completely free for non-commercial use

That sounds right. We can just charge for non-commercial use of the diffserver that matters, cost-wise. "First 10k free" seems like a fine start.